warp/tests/locality/test_cyclic.py

```
@@ -0,0 +1,51 @@
+
+refC = wp.zeros((nx,ny), dtype=float)
+wp.launch_tiled(copy, dim=(nx//BLOCKSIZE, ny//BLOCKSIZE), inputs=[A], outputs=[refC], device="cuda:0", block_dim=32)
+print(refC.numpy())
+
+# Use cyclic policy for round-robin distribution
+# dim is in terms of tiles: (nx//BLOCKSIZE, ny//BLOCKSIZE) = (6, 4)
+# places (2, 2) means 2x2 grid
+result = wp.cyclic(dim=(nx//BLOCKSIZE, ny//BLOCKSIZE), places=(2, 2))
+#result = wp.cyclic(dim=(nx//BLOCKSIZE, ny//BLOCKSIZE), places=1)
+
+print(f"\nCyclic policy result:")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)